packages/terra-application/src/application-navigation/ApplicationNavigation.jsx

```diff
@@ -126,6 +126,10 @@ const propTypes = {
    * An ApplicationNavigationWorkspace element and it's associated ApplicationNavigationWorkspaceItems.
    */
   workspace: PropTypes.element,
+  /**
+   *  A configuration object to render a utility button.
+   */
+  userActionConfig: userActionConfigPropType,
```

Comment:
Can we move this prop up to keep the prop list alphabetize?

Reply:
Done [374d274](https://github.com/cerner/terra-application/pull/305/commits/374d2742dd8d6324aedbb27942023a5584700f77)